src/indexer/stamper.rs

```diff
@@ -60,12 +61,43 @@ impl Stamper {
     pub fn stamp(&self) -> u64 {
         self.0.fetch_add(1u64, Ordering::SeqCst) as u64
     }
+
+    /// Given a desired count `n`, `stamps` returns an iterator that
+    /// will supply `n` number of u64 stamps.
+    pub fn stamps(&self, n: u64) -> MultiStamp {
```

Comment:
Could we simply return a `std::ops::Range`?

Reply:
Absolutely! I had not considered `Range` here. I'll make the changes tonight.

Reply:
Cool